WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionSummaryViewModel.cs

```diff
@@ -55,14 +55,14 @@ public void UpdateTransaction(BuildTransactionResult transactionResult, Transact
 
 		ConfirmationTimeText = $"Approximately {TextHelpers.TimeSpanToFriendlyString(info.ConfirmationTimeSpan)} ";
 
-		var destinationAmount = _transaction.CalculateDestinationAmount().ToDecimal(MoneyUnit.BTC);
-		var btcAmountText = $"{destinationAmount} BTC ";
+		var destinationAmount = _transaction.CalculateDestinationAmount();
+		var btcAmountText = $"{destinationAmount.ToFormattedString()} BTC ";
 		var fiatAmountText =
-			destinationAmount.GenerateFiatText(_wallet.Synchronizer.UsdExchangeRate, "USD");
+			destinationAmount.ToDecimal(MoneyUnit.BTC).GenerateFiatText(_wallet.Synchronizer.UsdExchangeRate, "USD");
 		AmountText = $"{btcAmountText}{fiatAmountText}";
 
 		var fee = _transaction.Fee;
-		var feeText = fee.ToFeeDisplayUnitString();
+		var feeText = $"{fee.ToFormattedString()} BTC ";
```

Comment:
This is broken now. If you take a look at the `ToFeeDisplayUnitString()` method, it returns the string in BTC or Sat depending on the UiConfig.

Reply:
ahh i guess i should've fixed that method instead... alright